posts/2021-05-07-antora-docs.adoc

```diff
@@ -28,30 +28,37 @@ Antora is designed on the premiss that writers want to spend their time writing,
 
-Antora treats https://docs.antora.org/antora/2.2/component-versions/#branches-as-versions[each version of the docs as a distinct branch] in the Git structure. Each new branch is created from the existing reference of the previous version so that only the updated content changes when a new branch is added. This approach makes it easy for our team to compare and manage the different versions of a doc. And for our users, moving between doc versions is as simple as selecting the relevant version from the version picker at the beginning of our sidebar navigation.
+Antora treats https://docs.antora.org/antora/2.2/component-versions/#branches-as-versions[each version of the docs as a distinct branch] in Git.  Each new branch is created from the existing reference of the previous version so that only the updated content changes when a new branch is added. This approach makes it easy for our team to compare and manage the different versions of a doc. And for our users, moving between doc versions is as simple as selecting the relevant version from the version picker at the beginning of our sidebar navigation. Furthermore, the Antora framework provides flexibility with how different parts of the doc are versioned. We can easily integrate content that is not versioned with each release, such as Javadoc.
 
 
 image::img/blog/version-picker.png[screen capture of the version picker,align="center",caption="The version picker"]
 
-=== A vibrant community
-Thanks to the vibrant and helpful Antora community, we've received plenty of help along our journey and look forward to contributing back, whether with code or by answering other users' questions. The beauty of an open source project like Antora (and Open Liberty) is that nothing happens in a bubble and all participants can benefit from https://en.wikipedia.org/wiki/Network_effect#Technology_lifecycle[the network effect]. New capabilities and bug fixes are added with https://gitlab.com/antora/antora[each Antora release]. We are happy to pass these benefits along to our users and share our own experience with the Antora community.
+=== Flexible customizations
+Another big draw for Antora is the ability to customize the framework to meet the needs of our documentation. Our docs mix together concept and task topics that aren written by our team with Javadoc and  reference pages that are generated directly from the Open Liberty source code. Rather than The line between these two types of content is not always definite. We enhance our generated content with manually written examples and explanations. Content from multiple repositories can appear in the same section of our navigation, such as the overview topics in our reference section. The Antora multi-repository framework was flexible enough for us to design our own solution for integrating all this content, without having to build everything from scratch.
```

Comment:
extra space in "with Javadoc and  reference pages"

Reply:
"Rather than The line between these two types of content is not always definite." needs fixing

Reply:
Thanks for the catch, fixed both